02-peer-protocol.md

```diff
@@ -914,21 +978,24 @@ This message initiates the v2 channel establishment workflow.
    * [`point`:`delayed_payment_basepoint`]
    * [`point`:`htlc_basepoint`]
    * [`point`:`first_per_commitment_point`]
-   * [`byte`:`channel_flags`]
    * [`opening_tlvs`:`tlvs`]
 
-
 1. `tlv_stream`: `opening_tlvs`
 2. types:
-   1. type: 1 (`option_upfront_shutdown_script`)
+   1. type: 0 (`upfront_shutdown_script`)
    2. data:
-       * [`u16`:`shutdown_len`]
-       * [`shutdown_len*byte`:`shutdown_scriptpubkey`]
-
+       * [`...*byte`:`shutdown_scriptpubkey`]
+   1. type: 1 (`channel_type`)
```

Comment:
I've added this one which is on `master`, but the dual funding PR really should be rebased on top of `master` to also inherit the requirements.